src/sentry/static/sentry/app/views/installWizard.jsx

```diff
@@ -163,7 +163,7 @@ const InstallWizard = React.createClass({
     let loadingIndicator = IndicatorStore.add(t('Saving changes..'));
 
     // We only want to send back the values which weren't disabled
-    let data = _.mapObject(
+    let data = _.mapValues(
       _.pick(options, option => !option.field.disabled),
```

Comment:
If so, please change (and good catch). Let's also test this page (I definitely did not, sigh).